docs/specifications/tests/Methods.md

```diff
@@ -1,69 +1,280 @@
-# Methods common to Test Specifications
+# Methods common to Test Case Specifications
 
-This is a list of generic methods used in many test specifications. The
-test cases that makes use of any of these methods should link directly to
+This is a list of generic methods used in many Test Case specifications. The
+Test Cases that makes use of any of these methods refer directly to
 this text.
 
-### Method 1: Obtain the parent domain
+## Method 1: Parent zone
 
-This metod tries to obain the parent domain from the domain used as input
-to this method.
+The method has been removed. The function is integrated in [BASIC01] instead.
 
-1. A recursive SOA-record lookup for the child domain name starting at the
-   root domain should be done, and the steps of the process recorded.
-2. If the recursion reaches a name server that responds with a redirect
-   directly to the requested domain, including functional glue, the test
-   succeeds. The domain through which the name server was found is
-   considered the parent domain.  
-3. If the recursion reaches a name server that authoritatively responds
-   with NXDOMAIN for the child domain, the test succeeds. The domain through
-   which the name server was found is considered the parent domain.
 
-### Method 2: Obtain "glue Name records" from parent
+## Method 2: Delegation name servers
 
-This method tries to obtain the authoritative name servers from the
-delegation of the parent domain.
+### Method identifier
+**METHOD2** Delegation name servers
```

Comment:
I think the change from **Method 2** to **METHOD2** is a degradation in terms of readability.